source/server/admin/admin.cc

```diff
@@ -338,8 +345,9 @@ Admin::HandlerPtr AdminImpl::findHandler(absl::string_view path_and_query,
         if (method != Http::Headers::get().MethodValues.Post) {
```

Comment:
Does it make sense making handlers_ a flat_hash_set?

Reply:
It wouldn't work as a flat_hash_set because it does prefix matching. It could be an rb_tree or std::map though. This iterative loop already exists though and I don't want to change it in this PR. I will leave a TODO